packages/react-search-ui-views/src/styles/themes/reference-ui/_base.scss

```diff
@@ -1,12 +1,12 @@
 html {
   box-sizing: border-box;
 }
-*, *:before, *:after {
+*,
+*:before,
+*:after {
```

Comment:
Since we're already here / linting stuff - pseudo classes should [prefer having two colons](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements#Syntax) instead of one to separate pseudo elements vs. pseudo classes (e.g. `:hover`). It might be nice to fix this to `::before` and `::after` 😄

Reply:
I agree. Although I'd prefer having something like "stylelint --fix" doing this for us :smile:. I'll keep it in mind for the future.

Reply:
Yeah! Strong agree to having stylelint+prettier doing this kind of thing automatically for us ✨

Reply:
I'm not going to fix that in this PR, but I do like your suggestion.

Reply:
Cool, no worries. :)